components/Header/Header.module.scss

```diff
@@ -31,22 +37,92 @@
   }
 }
 
+.containerHeader {
+  display: flex;
+  flex-direction: column;
+}
+
+.containerNav {
+  display: flex;
+  padding: 0 24px;
+  justify-content: space-between;
+}
+
+$color1: #8DC8E9;
+$color2: #8C5AE3;
```

Comment:
Name var in a semantic way. Same as for the mixins, var are considered to be reusable. What if there are other vars named "color1" in the code, will you be sure that all of them are the same?